WordPress/gutenberg
# TextControl: Set Storybook control types on help, label and type

## What?
- Set control type on `help`, `label` and `type` in the storybook for the `TextControl` component. 
- Fix example not showing in the story docs.

## Why?
- The story breaks if you try to add a `help` or `label` prop. 
- The example is not showing in the story.

## How?
- Set control type to `text` on `help`, `label` and `type` props.
- Remove `@example` from the example to make it visible in the docs.

## Testing Instructions
Open the `TextControls` story and make sure you can set `label`, `help` and `type` props without the story breaking

## Screenshots or screencast 
| Before | After |
|--------|-------|
| ![bug](https://user-images.githubusercontent.com/1415747/198895482-5a3256ca-438f-4b02-8287-01777eca604b.gif) | ![fixed](https://user-images.githubusercontent.com/1415747/198895521-efba5f88-c21f-4605-8ac5-b371e017b314.gif) |
